Copyright 2025 Philipp Stephani

Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at

    https://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.

Workaround for https://github.com/bazel-contrib/rules_python/issues/2762.

--- python/private/python_bootstrap_template.txt
+++ python/private/python_bootstrap_template.txt
@@ -438,6 +438,7 @@
   with open(rcfile_name, "w") as rcfile:
     rcfile.write(f'''[run]
 relative_files = True
+disable_warnings = module-not-imported, no-data-collected
 source =
 \t{source}
 ''')
@@ -470,10 +470,10 @@
     kparams['stdout'] = sys.stderr
     kparams['stderr'] = sys.stderr
 
-  ret_code = subprocess.call(
+  subprocess.call(
     params,
     **kparams
-  ) or ret_code
+  )
 
   try:
     os.unlink(rcfile_name)
